### PR TITLE
chore: bump taxonomy version and other requirements

### PR DESCRIPTION
**Pull request by navinkarkera**
https://github.com/openedx/taxonomy-connector/commit/280ea962b846f0657def287caa6ef6c9ed4a91df

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @navinkarkera!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Review by sameenfatima78 — APPROVED** [review]

File: requirements/local.txt [hunk 1]
```diff
@@ -813,9 +813,9 @@ ruamel-yaml==0.17.32
     # via semgrep
 ruamel-yaml-clib==0.2.7
     # via ruamel-yaml
-s3transfer==0.6.2
+s3transfer==0.7.0
     # via boto3
-selenium==4.12.0
+selenium==4.13.0
```
Comment (Ali-D-Akbar) [inline requirements/local.txt:818]: Tests are failing due to the recent selenium update. A PR is on it's way to pin selenium and also update taxonomy version so this PR may not be needed anymore: https://github.com/openedx/course-discovery/pull/4134

**Review by Ali-D-Akbar — COMMENTED** [review]
Can be closed once https://github.com/openedx/course-discovery/pull/4134 is merged.

File: requirements/local.txt (hunk 1 above)
Comment (navinkarkera) [reply requirements/local.txt:818]: @Ali-D-Akbar Thanks, I'll close this one.

**Comment by openedx-webhooks** [comment]
@navinkarkera Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/course-discovery&entry.867055334=https://github.com/openedx/course-discovery/pull/4135&entry.1484655318=https://github.com/navinkarkera&entry.752974735=2023-09-28+10:24&entry.1917517419=2023-09-28+13:56&entry.2133058324=No) so we can improve your experience in the future.

**Pull request closed without merging**